src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -572,6 +560,10 @@ Where:
 
 ### The Transport Client protocol
 
+{{#> callout type='warning' }}
```

Comment:
remove completely the transport client protocol section

Reply:
Then I could simplify the `## Configuring Nuxeo to Access the Elasticsearch Cluster`
by not mentioning the transport protocol, remove the `### The REST Client (default)` title and just keep the content directly as THE way to do things in nuxeo. WDYT ?

Reply:
yes please remove all ref to transport client, only rest client is supported